lms/djangoapps/instructor/views/api.py

```diff
@@ -1246,7 +1246,7 @@ def get_students_features(request, course_id, csv=False):  # pylint: disable=red
         query_features = [
             'id', 'username', 'name', 'email', 'language', 'location',
             'year_of_birth', 'gender', 'level_of_education', 'mailing_address',
-            'goals'
+            'city', 'country', 'goals'
```

Comment:
Sure. But note that this query_features array does not determine the order of the columns in the Excel document. The values of PROFILE_FEATURES in instructor_analytics/basic.py do.